cinema/views.py

```diff
@@ -57,7 +63,7 @@
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` are commented out. Ensure this is intentional, as it might leave the endpoints unprotected. Consider enabling them for production to secure the API.